src/fonduer/utils/data_model_utils/tabular.py

```diff
@@ -425,6 +439,11 @@ def _get_axis_ngrams(
     mention, axis, attrib="words", n_min=1, n_max=1, spread=[0, 0], lower=True
 ):
     span = _to_span(mention)
+
+    if not span.sentence.is_tabular():
+        yield None
+        return
```

Comment:
Looking at your test case, it looks like this should either yeild ngrams, or a single `[None]`, is that correct? Is

```py
yield None
return
```

different than

```py
return None
```

?

Reply:
1) Yes, that's the intended behavior. 
2) Also yes, replacing both lines with `return None` would result in `list(get_row_ngrams(mention))` being an empty list `[]` instead of `[None]` for a non-tabular mention.

Reply:
Got it.